### DISCUSSION TITLE
The power on self start function cannot be turned on on the car system

### DISCUSSION THREAD

**Discussion by wang0251** [rustdesk/rustdesk#5604] [Feature Request]
prompting "Multimedia does not support this operation"

**Comment by rustdesk** [comment]
First time know some one use RustDesk on car. I do not even own a gasoline car, :(

**Reply by wang0251** [reply]
The car system is equivalent to an Android tablet, Without "ignore Battery Optimizations" this feature . Hope the app can automatically enable services when running.
